### ISSUE TITLE
README example for raise_error_excludes throws error

### ISSUE THREAD

**Issue by johnthagen** [manuzhang/mkdocs-htmlproofer-plugin#12]
Configuring the plugin as described in the README:

```yaml
plugins:
    - search
    - htmlproofer:
        raise_error: True
        raise_error_excludes: 
          504: ['https://www.mkdocs.org/']
          404: ['https://github.com/manuzhang/mkdocs-htmlproofer-plugin']
          400: ['*']
```

Produces an error during build:

```
INFO    -  Building documentation... 
ERROR   -  Config value: 'plugins'. Error: Plugin value: 'raise_error_excludes'. Error: Expected type: <class 'list'> but received: <class 'dict'>
```

**Comment by johnthagen** [comment]
This might just be that #10 is not released yet?

**Comment by manuzhang** [comment]
Correct.

**Comment by manuzhang** [comment]
I've just released 0.2.0 which is aligned with the README now. Please check it out.
